IBaseEntity/IBaseEntity.h

```diff
@@ -79,29 +79,44 @@ interface IBaseEntity {
 
     // A collection of references to all BaseEntities present in the world
     // at any given moment
+    // NOTE: This information cannot be retrieved from a Blueprint since
+    // Blueprints cannot read IBaseEntities
     static TArray<IBaseEntity*> s_aBaseEntities;
 
+    // bool whether entities have already been initialized (e.g. PreInits and
+    // PostInits called)
+    static bool s_bHasAlreadyInitializedBaseEntities;
+

```

Comment:
We need to know if the game has already started so we can call `PreInit` and `PostInit` on actors that are spawned during the game